docs/remote/devcontainer-cli.md

```diff
@@ -0,0 +1,54 @@
+---
+Order: 12
+Area: remote
+TOCTitle: devcontainer CLI
+PageTitle: Installing and working with the devcontainer CLI
+ContentId: 8946213d-716e-41ca-955f-944a41c70353
+MetaDescription: Documentation on using the devcontainer CLI with the Visual Studio Code Remote - Containers extension
+---
+# `devcontainer` Command Line Interface
+
+Remote-Containers has a `devcontainer` Command Line Interface (CLI) which allows you to interact with a dev container from your terminal.
+
+## Installing the devcontainer CLI
+
+To install the `devcontainer` CLI:
+
+1. Ensure you have the latest version of the [Remote-Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) installed (must be at least `v0.188.0`)
+2. Launch Visual Studio Code and select  **Remote-Containers: Install devcontainer CLI** from the Command Palette (`kbstyle(F1)`)
+
+    ![The "Install `devcontainer` CLI" command](images/devcontainer-cli/install.png)
+
+    * **Windows**: You will be prompted to automatically add the `devcontainer` CLI to your `PATH` or to copy the `devcontainer` CLI path to your clipboard for you to add to your `PATH`.
+    * **macOS/Linux**: If the extension detects a `bin` folder ( or `.local/bin` folder) in your user home folder and in your `PATH` then you will have the option of adding a symlink to the `devcontainer` CLI to this location. You will also have the option to copy the `devcontainer` CLI path to your clipboard for you to add to your `PATH`.
+
+3. From a system shell (one not inside Visual Studio Code), run `devcontainer --help` to test the installation and see the CLI's built-in help. Note that you may need to restart your shell for `PATH` changes to take effect.
```

Comment:
What do we mean by system shell here? An external terminal?

Reply:
It could be a nice touch to include a screenshot of the `devcontainer ---help` commands. Or if we think they may be changing frequently, we could leave it off if it feels too difficult to maintain/it would become more distracting than helpful :)

Reply:
As I continued reading through the doc, I realized a list of commands (perhaps even a text box / markdown table rather than screenshot) could be helpful so that I know what I can do with the CLI going into this.

Reply:
Changed the wording for system shell
Added a code block with the `devcontainer --help` output - that feels like it would be simpler to maintain but hopefully still fulfill the intended purpose?

Reply:
I really like it! I think text is easier to maintain too, and showing users upfront what the commands should be is nice in case they don't get the expected output or they are just reading for now and not following along live.